lib/lock.js

```diff
@@ -170,8 +170,17 @@ Lock.prototype.extend = function(time, fn) {
   var key    = this._key;
   var shaval = this._shavaluator;
 
+  if (!fn && typeof time === 'function') {
+    fn = time;
```

Comment:
If somebody makes a mistake and calls expire with a callback but no time, the error returns in the shape of a promise because the argument fn is undefined. It just felt better to enforce the time to be passed as argument and get the error in the callback in case it wasn't.